packages/app-bridge/src/AppBridgeTheme.ts

```diff
@@ -114,13 +120,30 @@ export class AppBridgeTheme {
     }
 
     public async createLibrary(library: DocumentLibraryCreate) {
+        const projectId = library.settings?.project ?? (await this.#createProjectIdForLibrary(library));
+
         return createDocument<DocumentLibrary>({
             ...library,
             portalId: this.getPortalId(),
-            settings: { project: library.settings?.project ?? this.getProjectId() },
+            settings: { project: projectId },
         } as DocumentLibrary);
     }
 
+    async #createProjectIdForLibrary(library: DocumentLibraryCreate) {
```

Comment:
Why the # before function name?

Reply:
As it should be a private method in class.

Reply:
Please use `private` to align with the rest of the code.